README.md

```diff
@@ -52,38 +52,38 @@ General [Setup instructions](https://docs.komodoplatform.com/notary/setup-Komodo
+CCL | [komodo](https://github.com/komodoplatform/komodo) | [db5b016](https://github.com/KomodoPlatform/komodo/tree/db5b016cf7b8ffffe440ad774dee8c6815ce5726) | [![dPOW Status](https://badges.komodo.live/svg/CCL_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
 CHIPS | [chips](https://github.com/chips-blockchain) | [70362d](https://github.com/chips-blockchain/chips/commit/70362d54645748194b72e1cc753cca8be4f8629b) | [![dPOW Status](https://badges.komodo.live/svg/CHIPS_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-3P
-COQUICASH | [komodo](https://github.com/komodoplatform/komodo) | [e159b4e](https://github.com/KomodoPlatform/komodo/tree/e159b4e7a40d3886519401c4074e957a1f9d42ba) | [![dPOW Status](https://badges.komodo.live/svg/COQUICASH_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
-CRYPTO | [komodo](https://github.com/komodoplatform/komodo) | [e159b4e](https://github.com/KomodoPlatform/komodo/tree/e159b4e7a40d3886519401c4074e957a1f9d42ba) | [![dPOW Status](https://badges.komodo.live/svg/CRYPTO_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
-DEX | [komodo](https://github.com/komodoplatform/komodo) | [e159b4e](https://github.com/KomodoPlatform/komodo/tree/e159b4e7a40d3886519401c4074e957a1f9d42ba) | [![dPOW Status](https://badges.komodo.live/svg/DEX_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
-EMC2 | [emc2](https://github.com/emc2foundation/einsteinium) | [a716ae8](https://github.com/emc2foundation/einsteinium/tree/a716ae80995e51b9a9d0bd5977a3463bbda4a17e) | [![dPOW Status](https://badges.komodo.live/svg/EMC2_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-3P
+COQUICASH | [komodo](https://github.com/komodoplatform/komodo) | [db5b016](https://github.com/KomodoPlatform/komodo/tree/db5b016cf7b8ffffe440ad774dee8c6815ce5726) | [![dPOW Status](https://badges.komodo.live/svg/COQUICASH_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
+CRYPTO | [komodo](https://github.com/komodoplatform/komodo) | [db5b016](https://github.com/KomodoPlatform/komodo/tree/db5b016cf7b8ffffe440ad774dee8c6815ce5726) | [![dPOW Status](https://badges.komodo.live/svg/CRYPTO_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
+DEX | [komodo](https://github.com/komodoplatform/komodo) | [db5b016](https://github.com/KomodoPlatform/komodo/tree/db5b016cf7b8ffffe440ad774dee8c6815ce5726) | [![dPOW Status](https://badges.komodo.live/svg/DEX_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
+EMC2 | [emc2](https://github.com/emc2foundation/einsteinium) | [70d7dc2](https://github.com/emc2foundation/einsteinium/tree/70d7dc2b94e0b275f026ae51fda2a23725929bfd) | [![dPOW Status](https://badges.komodo.live/svg/EMC2_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-3P
```

Comment:
EMC2 seems outdated - https://github.com/emc2foundation/einsteinium/tree/a716ae80995e51b9a9d0bd5977a3463bbda4a17e is latest